recipe/meta.yaml

```diff
@@ -12,11 +11,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 2
-  skip: True  # [win or py27 or py34 or (osx and py35)]
+  number: 0
+  skip: True  # [win or py27 or py37]
```

Comment:
Based on the Gitter discussion, it seems like `py37` just isn't defined.  You need to go with `py>=3.7` or similar.  But you can just remove the `or py37`, given the restriction below.